src/main/java/frc/robot/RobotContainer.java

```diff
@@ -25,13 +27,16 @@ public class RobotContainer {
   private final SwerveSubsystem swerveSubsystem = new SwerveSubsystem();
 
   private final AutonSubsytem autonSubsytem = new AutonSubsytem(swerveSubsystem);
-  private SimulationSubsystem simulationSubsystem;
+
+  private SimulationSubsystem simulationSubsystem = new SimulationSubsystem(swerveSubsystem);
```

Comment:
Revert this. The simulation subsystem is initialized later

Reply:
Done and fixed null pointer exception that comes with removing it.